src/CMakeLists.txt

```diff
@@ -328,6 +328,20 @@ if(WITH_MULTIPROCESS)
     $<TARGET_NAME_IF_EXISTS:bitcoin_wallet>
   )
   list(APPEND installable_targets bitcoin-node)
+
+  # bitcoin_ipc_test library target is defined here in src/CMakeLists.txt
+  # instead of src/test/CMakeLists.txt so capnp files in src/test/ are able to
+  # reference capnp files in src/ipc/capnp/ by relative path. The Cap'n Proto
+  # compiler only allows importing by relative path when the importing and
+  # imported files are underneath the same compilation source prefix, so the
+  # source prefix must be src/, not src/test/
```

Comment:
9378b30774d40c8a093c69d2b1807c9eea32455a: I think you need to wrap this in `if(BUILD_TESTS)`

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30510#discussion_r1757005855

Thanks, added `if(BUILD_TESTS)`